src/components/Game/Robot.tsx

```diff
@@ -165,7 +165,7 @@ export const Robot = ({
 				textAnchor="middle"
 				data-test-id="label"
 			>
-				{id}
+				{id.slice(0, 3)}
```

Comment:
**suggestion**: turn this into a `shortId()` function, to ensure that the same transformation is applied everywhere and gets propagated properly when the requirements for having short ids changes.

The function then also is a central place to document why this is needed.

Reply:
Not fully sure what you meant by the last sentence: "The function then also is a central place to document why this is needed". just add a description of why it is needed? 

I assume it would be appropriate to add a test for the function at least.

Reply:
> Not fully sure what you meant by the last sentence: "The function then also is a central place to document why this is needed". just add a description of why it is needed?

Something like this:

```typescript

/**
 * This is needed, because ...
 */
const shortId = (id: string): string => id.slice(0,3)
```

> I assume it would be appropriate to add a test for the function at least.

Yes!